modules/openxr/openxr_interface.cpp

```diff
@@ -666,6 +666,7 @@ Transform3D OpenXRInterface::get_camera_transform() {
 Transform3D OpenXRInterface::get_transform_for_view(uint32_t p_view, const Transform3D &p_cam_transform) {
 	XRServer *xr_server = XRServer::get_singleton();
 	ERR_FAIL_NULL_V(xr_server, Transform3D());
+	ERR_FAIL_INDEX_V_EDMSG(p_view, get_view_count(), Transform3D(), "view index outside bounds");
```

Comment:
Is it possible to set an invalid view index easily from the editor, which would warrant getting a message in the editor UI?
If not, then you shouldn't use the `_EDMSG` variant but only the `_MSG` one. The message should also be a complete sentence (with capital letter, and period).

```suggestion
	ERR_FAIL_INDEX_V_MSG(p_view, get_view_count(), Transform3D(), "View index outside bounds.");
```

Reply:
thanks, I'll fix it in the other function as well